.github/scripts/release-docs.sh

```diff
@@ -53,6 +53,11 @@ VERSION_STRING_REPLACEMENT="version = \"${CHANNEL_VERSION}\""
 awk -v REPLACEMENT="${VERSION_STRING_REPLACEMENT}" '{gsub(/version = \"edge\"/, REPLACEMENT); print}' docs/config.toml > docs/config.toml.tmp
 mv docs/config.toml.tmp docs/config.toml
 
+# In docs/config.toml, change version to VERSION instead of latest or older version
+VERSION_STRING_REPLACEMENT="version = \"${CHANNEL_VERSION}\""
```

Comment:
Could you use a different variable name than above please?

Reply:
For which variable? `CHANNEL_VERSION` or `VERSION_STRING_REPLACEMENT`?

Reply:
VERSION_STRING_REPLACEMENT. The pattern in this script seems to be to use a different variable name for each replacement.

Reply:
Sounds good, changed it now!